pkgs/development/compilers/pforth/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "28";
+  name = "pforth";
+  src = fetchFromGitHub {
+    owner = "philburk";
+    repo = "pforth";
+    rev = "9190005e32c6151b76ac707b30eeb4d5d9dd1d36";
+    sha256 = "0k3pmcgybsnwrxy75piyb2420r8d4ij190606js32j99062glr3x";
+  };
+
+  buildPhase = ''
+    cd build/unix && make all
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp pforth_standalone $out/bin/pforth
+  '';
+
+
+  meta = {
+    description = "Portable ANSI style Forth written in ANSI C";
+    homepage = http://www.softsynth.com/pforth/;
+    license = stdenv.lib.licenses.publicDomain;
+    platforms = stdenv.lib.platforms.all;
```

since we are building the pforth `build/unix/` directory, we should restrict it to unix:
```suggestion
    platforms = stdenv.lib.platforms.unix;
```